src/gameComponents/DefenseTokens/DefenseToken.java

```diff
@@ -0,0 +1,152 @@
+package gameComponents.DefenseTokens;
```

Comment:
Did you have a notion of what else would live in the package?

Reply:
Not even remotely. I saw Squadrons/Squadron in the file tree and just assumed this was the done thing around here

Reply:
Eh, let's ignore it.  I plan to have another object in the Squadrons pkg soon, and changing packages around is cheap.